pandas/core/groupby.py

```diff
@@ -333,6 +333,16 @@ def _set_grouper(self, obj, sort=False):
     def groups(self):
         return self.grouper.groups
 
+    _init_defaults = dict(key=None, level=None, freq=None, axis=0,
```

Comment:
in other context we call this ``_attributes`` (I think we do this kind of signature generation already in resample). pls try to find examples and make consistent.

Reply:
Ok, changed